mix.exs

```diff
@@ -34,7 +34,8 @@ defmodule Bitcoinex.MixProject do
       {:timex, "~> 3.1"},
       {:decimal, "~> 1.0 or ~> 2.0"},
       {:ex_doc, ">= 0.0.0", only: :dev, runtime: false},
-      {:benchee, ">= 1.0.0", only: :dev}
+      {:benchee, ">= 1.0.0", only: :dev},
+      {:binary, "~> 0.0.5"}
```

Comment:
I am curious as to why we'd add a dependency that will do a [pass through](https://github.com/comboy/elixir-binary/blob/master/lib/binary.ex#L23) to `:binary.bin_to_list/1`?

Seem's like we can remove the dependency and call `:binary.bin_to_list/1` directly, am I missing something?

Reply:
Thanks for the feedback. Updated. https://github.com/RiverFinancial/bitcoinex/compare/407bdae54377eef81df33e9e10117ccd34dc5fc9..eb8cf16d6b7a55b7a022d3e7803bdb6d96fde498